framework/src/testing/mod.rs

```diff
@@ -10,9 +10,9 @@
 //! netbricks = { version = "1.0", features = ["test"] }
 //! ```
 
-pub(crate) mod arbitrary;
```

Comment:
ah I thought that these were needed to be used within nf-tests, no?

Reply:
yeah, all needed types are exported internally through `crate::testing` and externally through `netbricks::testing`. sub mods don't need any visibility.